integrationtest/neo/node/request/GetAll.d

```diff
@@ -218,33 +226,36 @@ private scope class GetAllImpl_v0
     final public void handle ( Storage storage, RequestOnConn connection,
         Const!(void)[] msg_payload )
     {
-        this.storage = storage;
-        this.conn = connection.event_dispatcher;
+        try
+        {
+            this.storage = storage;
+            this.conn = connection.event_dispatcher;
 
-        // Read request setup info from client.
-        bool start_suspended;
-        this.conn.message_parser.parseBody(msg_payload, start_suspended);
+            // Read request setup info from client.
+            bool start_suspended;
+            this.conn.message_parser.parseBody(msg_payload, start_suspended);
 
-        // Send initial status code response.
-        this.conn.send(
-            ( conn.Payload payload )
-            {
-                payload.addConstant(RequestStatusCode.Started);
-            }
-        );
-        this.conn.flush();
-
-        // Now ready to start sending data from the storage and to handle
-        // control messages from the client. Each of these jobs is handled by a
-        // separate fiber.
-        this.writer = new Writer;
-        this.controller = new Controller;
+            // Now ready to start sending data from the storage and to handle
+            // control messages from the client. Each of these jobs is handled by a
+            // separate fiber.
+            this.writer = new Writer;
+            this.controller = new Controller;
 
-        if ( start_suspended )
-            this.writer.suspender.requestSuspension();
+            if ( start_suspended )
+                this.writer.suspender.requestSuspension();
 
-        this.controller.fiber.start();
-        this.writer.fiber.start();
-        this.request_event_dispatcher.eventLoop(this.conn);
+            this.controller.fiber.start();
+            this.writer.fiber.start();
+            this.request_event_dispatcher.eventLoop(this.conn);
+        }
+        catch (Exception e)
```

Comment:
I really wonder if this would after all be worth handling at a higher level. The only reason I can think of to not do that is that the exact code to send is defined per request (atm).

Reply:
It's a tricky situation. The reason I didn't want it there, is that perhaps different requests may define different ways how the errors are reported to the client (and it requires change of the request definitions to expect it anytime). I guess I could add the separate method that runs the handler, accepts the error code (or use the swarm defined, in which case we'll have to add one), but then it's the question how should we plug this with the existing request.

Perhaps we should just add the request code and send it, as, if the client is not aware of it, it will shut down with the protocol error, which is still better than the current situation?

Reply:
I suppose we can leave it down to each request for now, then see what common pattern develops. (I suspect all requests will end up doing the same, but we may be surprised.)